pkg/antctl/antctl.go

```diff
@@ -108,6 +110,7 @@ var CommandList = &commandList{
 		},
 		{
 			use:          "addressgroup",
+			aliases:      []string{"addressgroups", "ag"},
```

Comment:
adg to complement atg?

Reply:
Thanks but I think it is not necessary because we will have more subcommands in the future.